submissions/VolodymyrRutskyi/task_js_dom/main.js

```diff
@@ -0,0 +1,186 @@
+            в массиве элемента, которое удовлетворяет условию переданному 
+            в callback функции. В противном случае возвращается undefined.</p>
+            `
+        ]
+    },
+    {
+        name: 'Findindex',
+        image: 'img/findindex.jpg',
+        description: [
+            `<p>Метод findIndex() возвращает индекс в массиве, 
+            если элемент удовлетворяет условию проверяющей 
+            функции. В противном случае возвращается -1.</p>
+            `
+        ]
+    },
+    {
+        name: 'forEach',
+        image: 'img/foreach.jpg',
+        description: [
+            `<p>Метод forEach() выполняет указанную функцию
+             один раз для каждого элемента в массиве.</p>
+            `
+        ]
+    },
+    {
+        name: 'Includes',
+        image: 'img/includes.jpg',
+        description: [
+            `<p>Метод includes() определяет, содержит
+            ли массив определённый элемент, возвращая в 
+            зависимости от этого true или false.</p>
+            `
+        ]
+    },
+    {
+        name: 'Join',
+        image: 'img/join.jpg',
+        description: [
+            `<p>Метод join() объединяет все элементы 
+            массива (или массивоподобного объекта) в строку.</p>
+            `
+        ]
+    },
+    {
+        name: 'Map',
+        image: 'img/map.jpg',
+        description: [
+            `<p>Метод map() создаёт новый массив с результатом 
+            вызова указанной функции для каждого элемента массива.</p>
+            <p>Метод map() не изменяет данный массив, создает копию</p>
+            `
+        ]
+    },
+    {
+        name: 'Push',
+        image: 'img/push.jpg',
+        description: [
+            `<p>Метод push() добавляет один или более элементов 
+            в конец массива и возвращает новую длину массива.</p>
+            `
+        ]
+    },
+    {
+        name: 'Reverse',
+        image: 'img/reverse.jpg',
+        description: [
+            `<p>Метод reverse() на месте обращает порядок следования
+            элементов массива. Первый элемент массива становится
+            последним, а последний — первым.</p>
+            `
+        ]
+    },
+    {
+        name: 'Slice',
+        image: 'img/slice.jpg',
+        description: [
+            `<p>Метод slice() возвращает новый массив, 
+            содержащий копию части исходного массива.</p>
+            <p>Метод slice() не изменяет данный массив, создает копию</p>
+            `
+        ]
+    },
+    {
+        name: 'Some',
+        image: 'img/some.jpg',
+        description: [
+            `<p>Метод some() проверяет, удовлетворяет ли 
+            какой-либо элемент массива условию, заданному 
+            в передаваемой функции.</p>
+            `
+        ]
+    }
+];
+
+let main = document.querySelector('.main');
+let navList = document.querySelector('.nav__list');
+let content = document.querySelector('.content');
+
+document.addEventListener('load', nav());
```

Comment:
Variables must be `const` because they don't change.

Alternatively, you can write without parentheses 
`document.addEventListener('load', nav);`

Reply:
fixed